InnerEye/ML/run_ml.py

```diff
@@ -121,15 +132,20 @@ def log_metrics(val_metrics: Optional[InferenceMetricsForSegmentation],
 class MLRunner:
 
     def __init__(self,
-                 model_config: ModelConfigBase,
+                 model_config: Optional[DeepLearningConfig] = None,
                  azure_config: Optional[AzureConfig] = None,
+                 container: Optional[LightningContainer] = None,
```

Comment:
[NIT] re-order to have model config next to container (as in the doc order)

Reply:
I've changed the doc order instead. We have quite a few places where we call with positional arguments, rather than keyword.

Reply:
OK, changed my mine, made your suggested change.